src/OmniSharp.DotNetTest/Legacy/LegacyTestManager.cs

```diff
@@ -63,22 +71,42 @@ public override RunDotNetTestResponse RunTest(string methodName, string testFram
                 WorkingDirectory = WorkingDirectory,
                 CreateNoWindow = true,
                 UseShellExecute = false,
-                RedirectStandardOutput = false,
-                RedirectStandardError = false
+                RedirectStandardOutput = true,
+                RedirectStandardError = true
             };
 
             var testProcess = Process.Start(startInfo);
 
-            var results = new List<TestResult>();
+            var output = new StringBuilder();
+            var error = new StringBuilder();
+
+            testProcess.OutputDataReceived += (_, e) =>
```

Comment:
Do we know if this can return a partial line? (or line and a half?)

I need to go looking at the .net process model again... :face_with_head_bandage: